storage/range_test.go

```diff
@@ -2411,28 +2410,33 @@ func TestRangeDanglingMetaIntent(t *testing.T) {
 		t.Errorf("expected original descriptor %s; got %s", &origDesc, &rlReply.Ranges[0])
 	}
 
-	// Try 100 lookups with IgnoreIntents. Expect roughly 50/50.
+	// Try 100 lookups with IgnoreIntents. Expect to see each descriptor at least once.
 	rlArgs.IgnoreIntents = true
-	var origCount, newCount int
-	for i := 0; i < 100; i++ {
-		clonedRLArgs := gogoproto.Clone(rlArgs).(*proto.InternalRangeLookupRequest)
-		clonedRLArgs.Timestamp = proto.ZeroTimestamp
-		rlReply.Reset()
+	var origSeen, newSeen bool
+	timeout := 50 * time.Millisecond
```

Comment:
Based on past experience, I'd just make this 1 second. Those test machines can make a mockery of how many instructions you feel your CPU should be capable of within 50ms.